vulnerabilities/data_source.py

```diff
@@ -209,7 +200,7 @@ def batch_advisories(self, advisories: List[Advisory]) -> Set[Advisory]:
 
         while advisories:
             b, advisories = advisories[: self.batch_size], advisories[self.batch_size:]
-            yield set(b)
+            yield b
```

Comment:
Add TODO or ticket to revisit this cryptic code

Reply:
Also do not use incorrect type hints as in `Set[Advisory]`... nothing is better than wrong.